### PR TITLE
Designs for Request access email

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
**_WHAT DOES THIS PR DO?_**
- Request access email template 

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

**Review by Baalmart — COMMENTED** [review]
Thanks @BenjaminSsempala . Are we able to also re-do the unit tests for this function?
The unit test does not need to pass for the first time.

**Comment by BenjaminSsempala** [comment]
> Thanks @BenjaminSsempala . Are we able to also re-do the unit tests for this function? The unit test does not need to pass for the first time.

Thanks @Baalmart Let me add the unit tests too.

File: src/auth-service/utils/email.msgs.js [hunk 1]
```diff
@@ -14,17 +14,118 @@ module.exports = {
...
+                            </tr>
+                        </table>
+                    </td>
+                </tr>
+            </table>
+        </div>
+
+    </body>
+
+</html>`;
```
Comment (BenjaminSsempala) [inline src/auth-service/utils/email.msgs.js:128]: @Baalmart I plan to find a way to shrink this html message into a constant or variable we can simply call in some followup Prs.

**Comment by BenjaminSsempala** [comment]
> Thanks @BenjaminSsempala . Are we able to also re-do the unit tests for this function? The unit test does not need to pass for the first time.

@Baalmart I've added the unit tests for this PR, thank You

**Review by Baalmart — APPROVED** [review]
thanks @BenjaminSsempala

**Pull request merged**